deploy/charts/openebs-monitoring/templates/dashboards-json-configmap.yaml

```diff
@@ -1,4 +1,6 @@
-{{- if and (index .Values "kube-prometheus-stack" "grafana" "enabled") (index .Values "kube-prometheus-stack" "grafana" "customDashboardsEnabled") }}
+{{- if or (index .Values "kube-prometheus-stack" "grafana" "enabled") (.Values.openebsMonitoringComponents.enabled ) }}
+{{- if or (index .Values "kube-prometheus-stack" "grafana" "openebsDashboardsEnabled") (.Values.openebsMonitoringComponents.grafana.openebsDashboardsEnabled )}}
```

Comment:
n_: just keep this same can we rename `openebsMonitoringComponents` as `openebs-monitoring-addon`